src/main/java/net/sf/jabref/exporter/layout/LayoutHelper.java

```diff
@@ -371,83 +308,64 @@ private void parseField() throws IOException, StringIndexOutOfBoundsException
                     firstLetter = name.charAt(0);
                 } catch (StringIndexOutOfBoundsException ex) {
                     StringBuilder lastFive = new StringBuilder(10);
-                    for (StringInt entry : parsedEntries.subList(Math.max(0, parsedEntries.size() - 6), parsedEntries.size() - 1)) {
+                    for (StringInt entry : parsedEntries.subList(Math.max(0, parsedEntries.size() - 6),
+                            parsedEntries.size() - 1)) {
                         lastFive.append(entry.s);
                     }
-                    throw new StringIndexOutOfBoundsException(Localization.lang("Backslash parsing error near") + " \'" + lastFive.toString().replace("\n", " ") + '\'');
+                    throw new StringIndexOutOfBoundsException(Localization.lang("Backslash parsing error near") + " \'"
```

Comment:
To be checked: Is localization needed here? Should be only relevant if the exception message is shown somewhere in the UI.